core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -177,6 +177,9 @@ private TableProperties() {
   public static final String UPDATE_ISOLATION_LEVEL = "write.update.isolation-level";
   public static final String UPDATE_ISOLATION_LEVEL_DEFAULT = "serializable";
 
+  public static final String WRITE_METADATA_USE_RELATIVE_PATH = "write.metadata.use.relative-path";
```

Comment:
It is worth to comment out in the code that this property is immutable once the table is created.

Reply:
Done